deepchem/models/torch_models/tests/test_chemberta.py

```diff
@@ -136,3 +136,21 @@ def test_chemberta_load_weights_from_hf_hub():
     # new model's model attribute is an entirely new model initiated by AutoModel.load_from_pretrained
     # and hence it should have a different identifier
     assert old_model_id != new_model_id
+
+
+@pytest.mark.hf
+def test_chemberta_finetuning_multitask_classification():
```

Comment:
please add a test for multi-task regression finetuning as well

Reply:
Added